shapepipe/modules/merge_starcat_package/merge_starcat.py

```diff
@@ -533,76 +539,96 @@ def process(self):
-            starcat_j = fits.open(name[0], memmap=False)
+            try:
+                starcat_j = fits.open(name[0], memmap=False, ignore_missing_simple=True)
+            except OSError as e:
+                print(f"Error while opening file '{name[0]}'")
+                #raise
+                continue
 
             data_j = starcat_j[self._hdu_table].data
 
             # positions
-            x += list(data_j['X'])
-            y += list(data_j['Y'])
-            ra += list(data_j['RA'])
-            dec += list(data_j['DEC'])
+            x += list(data_j["X"])
+            y += list(data_j["Y"])
+            ra += list(data_j["RA"])
+            dec += list(data_j["DEC"])
 
             # shapes (convert sigmas to R^2)
-            g1_psf += list(data_j['E1_PSF_HSM'])
-            g2_psf += list(data_j['E2_PSF_HSM'])
-            size_psf += list(data_j['SIGMA_PSF_HSM']**2)
-            g1 += list(data_j['E1_STAR_HSM'])
-            g2 += list(data_j['E2_STAR_HSM'])
-            size += list(data_j['SIGMA_STAR_HSM']**2)
+            g1_psf += list(data_j["E1_PSF_HSM"])
+            g2_psf += list(data_j["E2_PSF_HSM"])
+            size_psf += list(data_j["SIGMA_PSF_HSM"] ** 2)
+            g1 += list(data_j["E1_STAR_HSM"])
+            g2 += list(data_j["E2_STAR_HSM"])
+            size += list(data_j["SIGMA_STAR_HSM"] ** 2)
 
             # flags
-            flag_psf += list(data_j['FLAG_PSF_HSM'])
-            flag_star += list(data_j['FLAG_STAR_HSM'])
+            flag_psf += list(data_j["FLAG_PSF_HSM"])
+            flag_star += list(data_j["FLAG_STAR_HSM"])
 
             # misc
-            mag += list(data_j['MAG'])
-            snr += list(data_j['SNR'])
-            psfex_acc += list(data_j['ACCEPTED'])
+
+            # MKDEBUG: The following columns do not exist (yet)
+            # for psf converted (pix2wcs) files.
+            try:
+                mag += list(data_j["MAG"])
+            except:
+                mag += list(np.zeros_like(data_j["X"]))
+            try:
+                snr += list(data_j["SNR"])
+            except:
+                snr += list(np.zeros_like(data_j["X"]))
+            try:
+                psfex_acc += list(data_j["ACCEPTED"])
+            except:
+                psfex_acc += list(np.zeros_like(data_j["X"]))
 
             # CCD number
-            ccd_nb += [
-                re.split(r"\-([0-9]*)\-([0-9]+)\.", name[0])[-2]
-            ] * len(data_j['RA'])
+            ccd_nb += [re.split(r"\-([0-9]*)\-([0-9]+)\.", name[0])[-2]] * len(
+                data_j["RA"]
+            )
 
         # Prepare output FITS catalogue
+        # MKDEBUG: SEx_cat=True -> False
         output = file_io.FITSCatalogue(
-            f'{self._output_dir}/full_starcat-0000000.fits',
+            f"{self._output_dir}/full_starcat-0000000.fits",
             open_mode=file_io.BaseCatalogue.OpenMode.ReadWrite,
-            SEx_catalogue=True
+            SEx_catalogue=False,
```

Comment:
Propagating information from original catalogue (``True``) is not needed, and moves the main data to HDU #2, not compatible with other files.